.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "starcache"]
+	path = starcache
+	url = https://github.com/StarRocks/starcache.git
```

Comment:
Need to depend on a specified version?

Reply:
Good to know, how to achieve it?

Reply:
> Good to know, how to achieve it?

Every time we manually pull the starcache submodule, the commit id will be updated to files 
 in starrocks/.git/module/starcache directory, such as the `HEAD` file.